Automattic/simplenote-android
# Add a script to update metadata for translation

This PR adds a lane to support the update of the .pot file that is uploaded to GlotPress.
This PR builds on top of https://github.com/Automattic/simplenote-android/pull/745 and should be tested and merged after 745 is merged.

# To test
1. Run `bundle install`.
2. Checkout a new branch out of this one.
3. Make changes in some of the `txt` files.
4. Run `bundle exec fastlane update_appstore_strings version:1.9` and verify that new changes are applied to `PlayStoreStrings.pot`.

### Clean up
5. Delete the new branch.
